docs/NewFlux_Comentary.md

```diff
@@ -0,0 +1,252 @@
+# New Flux
+
+Author: Nathaniel Cook
+
+Recently Paul Dix and I added a document on the vision of a new Flux that is more usable.
+The design there solves several key points of feedback we have received on Flux.
+
+
+That document describe the what and some of the why. This document will discuss the how.
+This document is not meant to be prescriptive rather just my personal commentary on how I see the vision of new Flux could be implemented.
+It is meant to be a seed to start a good team discussion around how  we can implement the new Flux vision.
+
+
+## What is new?
+
+First before we talk about how we would implement the new vision, we should clarify what is new.
+
+### Single Table vs Stream of Tables
+
+The new Flux data model changes to be a single table with a homogeneous schema.
+This is different from the current design which represents data as a stream (unbounded sequence) of tables with possibility differing schemas.
+
+This change is fundamental to the new design, the most common pit fall with current Flux is working with this stream of tables that may or may not be the same structure.
+
+### Influxdb.From is always pivoted
+
+To go along with the single table and homogeneuos schema the data returned from the `influxdb.from` function will be _pivoted_.
+Specifically the _\_field_ and _\_value_ structure of the data has been removed and instead column names correspond with the name of the InfluxDB tag or field.
+Like before no distinction is made between tags and fields as columns in the data model.
+
+A consequence of this change is that the `from` call now also requires the `measurement` parameter as the InfluxDB data model only guarantees a homogeneuos schema within the scope of a measurement.
+
+### Syntax Sugar for From
+
+An additional syntax is proposed that makes querying data from InfluxDB much easier to type.
+
+The syntax is:
+
+```
+@bucket.measurement
+```
```

Comment:
I really like the simplicity of this syntax. I feel like it will be much for familiar to user's coming from other scripting languages.